api/users.rb

```diff
@@ -64,6 +64,8 @@
 end
 
 put "#{APIPREFIX}/users/:user_id" do |user_id|
+  Mongoid::Clients.default.close
+  Mongoid::Clients.default.reconnect
```

Comment:
Alternatively, we can force reconnections when writing to MongoDB so we can use the commented out solution below, where it allow for staying connected to a secondary, just reconnecting to a primary to prevent write failures.